pkg/cloud/network.go

```diff
@@ -132,14 +133,14 @@ func (c *client) CreateIsolatedNetwork(csCluster *capcv1.CloudStackCluster) (ret
 	setIfNotEmpty(csCluster.Status.DomainID, p.SetDomainid)
 	resp, err := c.cs.Network.CreateNetwork(p)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "error encountered when creating network with name: %s", netStatus.Name)
```

Comment:
Putting wrapfs into place in a lot of places as the Cloudstack error messages aren't really enough context.